### DISCUSSION TITLE
Is it possible to set tina/config.js to render markdown in double quotes?

### DISCUSSION THREAD

**Discussion by wristtattoo** [tinacms/tinacms#4117] [Help]
I'd like my markdown to render in my NextJS (.js not .ts) project as `title: "This is my title wrapped in double quotes"` instead of it currently being rendered without as `title: This is my title without any quotes wrapping it`

Is there something i should add in the schema to achieve this? Thanks

**Comment by TaylanTatli** [comment]
I keep looking for this. My Next.js website not working without surrounding date with quotes and I don't know how to fix this.

**Reply by TaylanTatli** [reply]
@wristtattoo did you find a solution?
